enterprise/server/remote_execution/runner/runner.go

```diff
@@ -513,11 +513,10 @@ func (r *taskRunner) RemoveInBackground(ctx context.Context) {
 func (r *taskRunner) isCIRunner() bool {
 	r.p.mu.RLock()
 	task := r.task
-	props := r.PlatformProperties
 	r.p.mu.RUnlock()
 
 	args := task.GetCommand().GetArguments()
-	return props.WorkflowID != "" && len(args) > 0 && args[0] == "./buildbuddy_ci_runner"
+	return len(args) > 0 && args[0] == "./buildbuddy_ci_runner"
```

Comment:
```suggestion
	return platform.IsCICommand(task.GetCommand(), task.GetCommand().GetPlatform())
```

Reply:
eh, nvm, that func is looking at some stuff related to the GH actions runner that I deleted. will clean that up later.